src/pages/home/Home.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import { CurrentWeather } from '../../components';
+import { publicApiInstance } from '../../utils/api';
+import endpoints from '../../config/enpoints';
+
+const Home = () => {
+  const [data, setData] = useState([]);
+  const [location, setLocation] = useState({
+    lon: 25.2657137,
```

Comment:
is it needed to define default state with some magic numbers?

Reply:
I choose default value because user have possibility not allow set current location.

Reply:
Ok, I will try to add message.